app/src/main/java/com/amazonaws/kinesisvideo/demoapp/fragment/StreamWebRtcConfigurationFragment.java

```diff
@@ -181,6 +186,19 @@ private void setRegionFromCognito() {
         }
     }
 
+    private String generateDualStackEndpoint(String region) {
```

Comment:
It might make more sense to have this as a static method in a different class (utils?)

Reply:
There aren't any existing files or directories that
I didn't think any of the existing classes nor directories in the app would be a better fit than here, would you prefer to have new class for this?

Reply:
New class is what I was thinking. KvsServiceClientFactory or something and we can move the client (configuration) creations to there as well.

Reply:
Should make it easier to replace clients in the future :)